src/hermes/commands/harvest/cff.py

```diff
@@ -47,13 +47,15 @@ def __call__(self, command: HermesHarvestCommand) -> t.Tuple[t.Dict, t.Dict]:
 
         # Validate the content to be correct CFF
         cff_dict = self._load_cff_from_file(cff_data)
+
         if command.settings.cff.enable_validation and not self._validate(cff_file, cff_dict):
             raise HermesValidationError(cff_file)
 
         # Convert to CodeMeta using cffconvert
         codemeta_dict = self._convert_cff_to_codemeta(cff_data)
         # TODO Replace the following temp patch for #112 once there is a new cffconvert version with cffconvert#309
         codemeta_dict = self._patch_author_emails(cff_dict, codemeta_dict)
+        codemeta_dict["version"] = str(codemeta_dict["version"])    # Convert Version to string
```

Comment:
Do we know for certain that the `version` field exists? 🤔 I would guard this with an `if "version" in codmeta_dict`

Reply:
> Do we know for certain that the `version` field exists? 🤔 I would guard this with an `if "version" in codmeta_dict`

No, we don't know that. It's not a required field in CFF.

Reply:
Should we leave a comment (maybe in a logger or so) that the version is required for some deposition platforms?

Reply:
Is it? On Invenio(-RDM) it defaults to 1.0

Reply:
ah ok then not^^